freqtrade/freqtradebot.py

```diff
@@ -433,11 +442,12 @@ def execute_buy(self, pair: str, stake_amount: float) -> bool:
         """
         pair_s = pair.replace('_', '/')
         pair_url = self.exchange.get_pair_detail_url(pair)
-        stake_currency = self.config['stake_currency']
-        fiat_currency = self.config.get('fiat_display_currency', None)
 
+        fiat_currency = self.config.get('fiat_display_currency', None)
+        
         # Calculate amount
         buy_limit = self.get_target_bid(pair, self.exchange.get_ticker(pair))
+        stake_currency = self.config['stake_currency']
```

Comment:
Is there a reason to move this by 5 lines?

it's not important - but also doesn't make sense to me ...

Reply:
there is no reason will move it back to top again